src/dinitctl.cc

```diff
@@ -324,8 +359,52 @@ int dinitctl_main(int argc, char **argv)
             cmdline_error = true;
         }
     }
+    else if (command == command_t::SIG_SEND) {
+        if (show_siglist) {
+            if (sigstr.empty()) {
+                command = command_t::SIG_LIST;
+            }
+            else {
+                cmdline_error = true;
+            }
+        }
+        else {
+            if (sigstr.empty()) {
+                cerr << "dinitctl: signal number/name must be specified" << std::endl;
+                return 1;
+            }
+            else if (sigstr == "NONE" | sigstr == "none") {
```

Comment:
It's not a good idea to use constants (like `"NONE"` and `"none"`) in multiple places without creating a named constant (variable). Because, if it gets changed in one place, there is no indication that it needs to be changed in the other place too.

There's no need to refer to "none" and "NONE" here though, we can check if `signal_name_to_number` returns 0 instead. That solves the problem much more cleanly.